.github/workflows/e2e-test-jsm.yaml

```diff
@@ -0,0 +1,58 @@
+name: E2E Testing JSM
```

Comment:
We already have two e2e tests (with/without domain). E2e tests are heavy and expensive. I think we should avoid adding more workload as much as possible. 

Maybe we can run one of tests, say E2e test without domain, using JSM. In this case we have involved JSM in integration test and in the same time did not add a new workload to our verification process and also Jira software is tested in E2e with domain test.

Reply:
yup, make sense. Moved jsm to the domain one.